internal/config/validate.go

```diff
@@ -110,3 +59,70 @@ func (c *Config) Validate(opts ValidationOpts) error {
 
 	return nil
 }
+
+// Validation for the `bktec run` command
+func (c *Config) ValidateForRun() error {
+	_ = c.validate()
```

Comment:
We can ignore the return from `validate()` here because the errors will continue to be accumulated on `c.errs` by the checks in `ValidateForRun` and then all returned at the end of the method.

The same applies to `ValidateForPlan`.

The return value of `validate()` could technically be removed but it would require a rework of all of the existing tests.